src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -176,8 +180,19 @@ public static function createConnection($dsn, array $options = [])
             $redis = new $class();
 
             $initializer = static function ($redis) use ($connect, $params, $dsn, $auth, $hosts) {
+                $host = $hosts[0]['host'] ?? $hosts[0]['path'];
+                $port = $hosts[0]['port'] ?? null;
+
+                if (isset($params['redis_sentinel'])) {
+                    $sentinel = new \RedisSentinel($host, $port, $params['timeout'], (string) $params['persistent_id'], $params['retry_interval']);
+
+                    if (!([$host, $port] = $sentinel->getMasterAddrByName($params['redis_sentinel']))) {
```

Comment:
the brackets are not needed just after the `!`

Reply:
PHPStorm complains about `Can't use temporary expression in write context`, but I can try

Reply:
Pushed :)